crates/chaintsn/src/errors.rs

```diff
@@ -58,6 +58,12 @@ pub enum OpError {
     #[error("op referenced non-existent deposit {0}")]
     UnknownDeposit(u32),
 
+    #[error("op referenced invalid deposit {deposit_idx}; operator idx: {operator_idx}")]
```

Comment:
Should be `op referenced invalid deposit (deposit {deposit_idx}, operator {operator_idx})`.